packages/protobuf/src/next/to-json.ts

```diff
@@ -151,7 +155,25 @@ function reflectToJson(msg: ReflectMessage, opts: JsonWriteOptions): JsonValue {
       json[jsonName(f, opts)] = jsonValue;
     }
   }
-  // TODO: Add support for extensions.
+  const descSet = opts.descSet;
+  if (descSet?.getExtensionFor) {
+    const tagSeen = new Set<number>();
+    for (const uf of msg.getUnknown() ?? []) {
+      if (tagSeen.has(uf.no)) {
```

Comment:
Can you add a comment that explains that tags can appear multiple times for a single extension field, and this saves us from evaluating them multiple times?